dd-trace-core/src/main/java/datadog/trace/common/writer/common/Payload.java

```diff
@@ -1,4 +1,4 @@
-package datadog.trace.common.writer.ddagent;
+package datadog.trace.common.writer.common;
```

Comment:
I probably would drop the second common here:
```suggestion
package datadog.trace.common.writer;
```